try-it-out/docs/pages/deploy-devops-tools.rst

```diff
@@ -0,0 +1,60 @@
+.. _Deploy Devops Tools:
+-----------------------
+Deploy The Devops Tools
+-----------------------
+
+In this section, you will be deploying all of the DevOps tools that we will be using within this tutorial. This consists of deploying a Jenkins instance, a SonarQube server, and a local (insecure) Docker registry. 
```

Comment:
If this is really the first experience someone will have with SDP - the may not know what those tools are. 

maybe a ``csv-table`` that outlines the tool and a *brief* description of what it is?